### DISCUSSION TITLE
is there something like rust default packages

### DISCUSSION THREAD

**Discussion by rubiin** [jdx/mise#2329] [General]
I was searching for something like .default-go-packages but for rust.

**Comment by jdx** [comment]
While it wouldn't be hard to add, the cargo backend is probably a better solution

**Reply by rubiin** [reply]
@jdx Just switched from asdf and I have been enjoying using mise so far.For now I am currently using a script to install the packages I need as a workaround . It would be great to have it I guess 
cc: https://github.com/code-lever/asdf-rust
